### PR TITLE
feat(www): add accessibility statement

### PR DESCRIPTION
**Pull request by marcysutton**
## Description

This PR adds a central place for Gatsby accessibility information to live on the .org site. It's currently linked from the homepage, with a centered text link at the bottom. We should add it to other sections too, but that will require some design work to fit a footer on pages that go full-height. This PR was intended to ship something we can link to, specifically for @amberleyromo's talk at JAMStack conf.

We'll want to update the accessibility statement whenever we have new information to add, particularly around our current status / commitment to WCAG 2.0 (and the last edited date on the statement...perhaps could be automated? seemed like overkill to me.)

EDIT: the homepage link was removed and #13221 was opened to sort out the design / placement.

### CODE REVIEW

**Comment by marcysutton** [comment]
@fk any design input on the impromptu homepage footer?

File: docs/accessibility-statement.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Third-party platforms, products and services
+
+Common Gatsby workflows involve third-party products and services, such as sourcing from Content Management Systems, and managing and deploying sites with hosting platforms. Some of these platforms appear to have been designed and/or built without accessibility in mind; we encourage those vendors to improve their accessibility along with us.
+
+Our current recommendations for accessible CMS platforms are:
+
+- Wordpress with the Classic Editor Plugin
+- Drupal
+
+For managing and deploying Gatsby sites, we recommend:
```
Comment (DSchau) [inline docs/accessibility-statement.md]: Is this based on accessibility reasons? If so, may want to mention that too, e.g.

```suggestion
For managing and deploying Gatsby sites with accessible tools, we recommend:
```

File: docs/accessibility-statement.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Gatsby products and services
+
+Gatsbyjs.org is the online home of the open source Gatsby website framework; it includes documentation, tutorials, and guides on how and why to build with Gatsby, as well as a blog, site showcase, and information on how to contribute to the project.
+
+Gatsbyjs.com is the business website for Gatsby, Inc. the startup building Gatsby, including our first product, Gatsby Preview: a cloud service allowing team members to collaborate on a Gatsby site in development.
+
+## Providing feedback and getting accessibility help
+
+We strive to make these websites and the Gatsby framework itself as accessible as possible. Our goal is to meet WCAG 2.0 AA, with which we are partially compliant. We’ll be the first to admit the Gatsby ecosystem is a work in progress, and we are open to all feedback to make things better.
```
Comment (DSchau) [inline docs/accessibility-statement.md]: Worth adding a link here?

```suggestion
We strive to make these websites and the Gatsby framework itself as accessible as possible. Our goal is to meet [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), with which we are partially compliant. We’ll be the first to admit the Gatsby ecosystem is a work in progress, and we are open to all feedback to make things better.
```

**Comment by marcysutton** [comment]
> In the interest of not blocking this PR--could we introduce the document, and then in a separate PR introduce the footer?

Yes we can. I'll remove the link with a commit and open an issue to sort out placement!

**Pull request merged**